src/rules/onlyArrowFunctionsRule.ts

```diff
@@ -25,9 +25,13 @@ export class Rule extends Lint.Rules.AbstractRule {
         ruleName: "only-arrow-functions",
         description: "Disallows traditional (non-arrow) function expressions.",
         rationale: "Traditional functions don't bind lexical scope, which can lead to unexpected behavior when accessing 'this'.",
-        optionsDescription: "Not configurable.",
+        optionsDescription: Lint.Utils.dedent`
+            One argument may be optionally provided:
+
+            * \`"allow-declarations"\` allows standalone function declarations.
+        `,
         options: null,
```

Comment:
We'll want to specify this now, you can follow this example:
https://github.com/palantir/tslint/blob/80683f627f52df28feceee9b6dd1b048b45c2a96/src/rules/oneVariablePerDeclarationRule.ts#L33L41